Source/WebCore/Modules/websockets/WebSocketHandshake.cpp

```diff
@@ -104,7 +104,7 @@ String WebSocketHandshake::getExpectedWebSocketAccept(const String& secWebSocket
     SHA1 sha1;
     CString keyData = secWebSocketKey.ascii();
     sha1.addBytes(keyData.span());
-    sha1.addBytes(webSocketKeyGUID, std::size(webSocketKeyGUID) - 1);
+    sha1.addBytes(std::span { webSocketKeyGUID, std::size(webSocketKeyGUID) - 1 });
```

Comment:
webSocketKeyGUID should be `std::array`, and we can call `span()` on it.

Reply:
Will fix, thank you!

Reply:
I couldn't get this to cleanly convert to `std::array` (or directly to `std::span`). The type inference makes this an array of `char[37]`, and `std::array` (or `span`) want you to pass it the start and length. There's probably a clever way to do this but I couldn't figure it out quickly.

Reply:
`std::to_array<uint8_t>()` may help. But it's OK to leave as is for now if you can't make it work.